test/controllers/api/docs_controller_test.exs

```diff
@@ -58,16 +52,59 @@ defmodule HexWeb.API.DocsControllerTest do
     assert conn.resp_body == "HEYO"
   end
 
+  @tag :integration
```

Comment:
Ah okay, it's fine to go to the remote, just more complicated to add code that skips it :)